hack/lib/sync-charts.sh

```diff
@@ -28,16 +28,6 @@ git config --global user.email "openyurt-bot@openyurt.io"
 git config --global user.name "openyurt-bot"
 git clone --single-branch --depth 1 git@github.com:openyurtio/charts.git charts
 
-echo "clear charts/yurt-coordinator in openyurtio/charts"
-
-if [ -d "charts/charts/yurt-coordinator" ]
-then
-    echo "charts yurt-coordinator exists, remove it"
-    rm -r charts/charts/yurt-coordinator/*
-else
-    mkdir -p charts/charts/yurt-coordinator
-fi
```

Comment:
I think we need to remove the yurt-coordinator folder in openyurtio/charts repo. so how about optimize this code snippet as following:

```
if [ -d "charts/charts/yurt-coordinator" ]
then
    echo "charts yurt-coordinator exists, remove it"
    rm -r charts/charts/yurt-coordinator
if
```